backend/app/main.py

```
@@ -0,0 +1,136 @@
+from fastapi import FastAPI, HTTPException
+from fastapi.middleware.cors import CORSMiddleware
+from .models import UserResponse, UserData
```

Ruff (0.12.2)

`.models.UserData` imported but unused

Remove unused import: `.models.UserData`

(F401)